pkg/collector/corechecks/servicediscovery/usm/python.go

```diff
@@ -114,13 +112,13 @@ func (p pythonDetector) findNearestTopLevel(fp string) string {
 }
 
 func (g gunicornDetector) detect(args []string) (ServiceMetadata, bool) {
-	if fromEnv, ok := extractEnvVar(g.ctx.envs, gunicornEnvCmdArgs); ok {
+	if fromEnv, ok := extractEnvVar(g.ctx.envs, "GUNICORN_CMD_ARGS"); ok {
```

Comment:
The changes in this file look unnecessary.

Reply:
this code is restored to original